File: Backjoon4344/Backjoon4344.c
#include <stdio.h>

int main(){
    int n[6] = {5,50,50,70,80,100};
    int Nn[5];
    float sum = 0.0, over = 0.0;
    for(int i = 1; i < n[0]; i++){
        sum += n[i];
        Nn[i-1] = n[i]; 
    }
    for(int i = 1; i < n[0]; i++){
        if(sum / n[0] < n[i])
            over += 1.0; 
    }
    printf("%f%%", over/n[0]*100);
    return 0;
}

File: Backjoon4344/Backjoon4344.java
package Backjoon4344;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Backjoon4344 {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int[] c = new int[1000];
		int C = Integer.parseInt(br.readLine());
        StringBuilder sb = new StringBuilder();
		StringTokenizer st;
		for(int i = 0 ; i < C; i++) {
			st = new StringTokenizer(br.readLine()); 
			int N = Integer.parseInt(st.nextToken());	
			c = new int[N];
			double sum = 0;	
			for(int j = 0 ; j < N ; j++) {
				c[j] = Integer.parseInt(st.nextToken());	
				sum += c[j];	
			}
			sum /= N;
			double count = 0; 
			for(int j = 0 ; j < N ; j++) 
				if(c[j] > sum) 
					count++;
            sb.append(String.format("%.3f",(count/N)*100)).append("%\n");
		}
        System.out.print(sb);
	}
}